zigzag/zigzag.py

```diff
@@ -37,12 +38,13 @@ def __init__(self,
         swagger_client.configuration.api_key['Authorization'] = qtest_api_token
         self._qtest_api_token = qtest_api_token
         self._junit_xml_file_path = junit_xml_file_path
-        self._qtest_project_id = qtest_project_id
-        self._qtest_test_cycle_name = qtest_test_cycle
         self._pprint_on_fail = pprint_on_fail
+        self._config_file = config_file
```

Comment:
I think you should create a property for config file to line up with other properties on this class.  Rather than accessing this internal attribute from outside this class.